packages/admin-ui/src/app/app.module.ts

```diff
@@ -60,6 +60,8 @@ import {BuiltinIamAppsListComponent} from "./security/builtin_iam/builtin-iam-ap
 import {BuiltinIamAppCreateComponent} from "./security/builtin_iam/builtin-iam-app-create.component";
 import {BuiltinIamAppDetailComponent} from "./security/builtin_iam/builtin-iam-app-detail.component";
 import {PlatformRoleCreateComponent} from "./security/roles/platform-role-create.component";
+import {DropZoneDirective} from "./participants/drop-zone.directive";
```

Comment:
This was moved already to _pipes_and_guards, please make sure to merge from main before submitting PRs